public_html/index.html

```diff
@@ -9,4 +9,54 @@
 			<img class="splash" src="assets/images/pic.jpg" width="700" height="auto" align="middle">
 		</div>
 	</body>
+
+	<div class="footer" style="margin-top:100px">
+	<p>Copyright  2016 - TNS. All Rights Reserved. Photos probably used by permission.</p>
+	<p>"Thorne & Spindle changed my life! How can I get frequent product updates?!"</p>
+	<table style="width:100%">
+		<tr>
+			<td>Site</td>
+			<td>About</td>
+			<td>Connect</td>
+		</tr>
+		<tr>
+			<td>
+				<ul align="center">
+					<li><a class="footer" href="link1.html">Link1</a></li>
+					<li><a class="footer" href="link2.html">Link2</a></li>
+				</ul>
+			</td>
+			<td>
+				<ul>
+					<li>test</li>
+				</ul>
+			</td>
+			<td>
+				<ul align="center">
+					<li><a href="https://twitter.com/justinbieber">
+		<img src="assets/images/twitter.png" style="margin-left:10px" width="20" height="auto">
+			</a></li>
+					<li><a href="https://www.facebook.com/hmtheus">
+		<img src="assets/images/facebook.png" style="margin-left:10px" width="20" height="auto">
+			</a></li>
+					<li><a href="http://i.imgur.com/E7gjfgu.png?1">
+		<img src="assets/images/instagram.jpg" style="margin-left:10px" width="20" height="auto">
+			</a></li>
+				</ul>
+			</td>
+		</tr>
```

Comment:
I think the tables are a bad idea, but Im kinda working on ripping off bootstrap and using their `row` and `col-md-4`

lets work on it tomorrow